policies/policy_service.go

```diff
@@ -357,4 +358,33 @@ func (s policiesService) InactivateDatasetByIDInternal(ctx context.Context, owne
 	}
 
 	return nil
+}
+
+func (s policiesService) DuplicatePolicy(ctx context.Context, token string, policyID string) (Policy, error) {
+
```

Comment:
I guees it should save duplicated policy on database. Probably receive at least the new name from the client or create a logic to generate an automatic name.